src/admiral/certs/tasks.py

```diff
@@ -46,8 +46,12 @@ def summary_by_domain(domain, subdomains=True, expired=False):
         f"https://crt.sh/?Identity={wildcard_param}{domain}{expired_param}"
         f"&output=json"
     )
-
-    req = requests.get(url, headers={"User-Agent": "cyhy/2.0.0"})
+    connect_timeout, read_timeout = 5.0, 30.0
```

Comment:
I added a comment in [8b53eb9](https://github.com/cisagov/admiral/pull/21/commits/8b53eb9bb477dd514449e6401bf54ee51d6c47ea) to explain that these are default values for Celery Tasks that perform web requests. I also redefined the timeout values as constants. I don't know if that is best practice or not, but I wanted to avoid duplicating the values and comment across both tasks.